content/changelog/2025-06-13.md

```diff
@@ -0,0 +1,30 @@
+---
+title: app.build available via Homebrew, Neon MCP Server updates, and more
+---
+
+## app.build — now available via Homebrew
+
+Last week, we introduced [app.build](https://www.app.build/), our open-source reference implementation for building AI-powered applications on top of Neon. Unlike LLMs that generate isolated snippets, `app.build` uses agent architecture to turn prompts into fully deployed, production-ready applications — complete with frontend, backend, and a Neon Postgres database.
+
+This week, in addition to installing `app.build` using `npx`, you can now install it using Homebrew:
+
+![brew app.build install](/docs/relnotes/brew_appdotbuild.png)
+
+📌 We also fixed an issue where newly built apps could flicker between the “Under Construction” page and the actual app. Apps now load consistently.
+ 
+## Neon MCP homepage & streamable HTTP support
+
+- **Neon MCP Server has a new homepage** — We’ve launched a new homepage for the Neon MCP Server at [mcp.neon.tech](https://mcp.neon.tech), making it easier to understand what the MCP Server does and what tools it supports.
```

Comment:
I think saying "now has a homepage" is better because we didn't have a homepage before.